content/articles/2018-12-pkgdown-1-3-0.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: 'pkgdown 1.3.0'
+author: Mara Averick
+date: '2018-12-10'
+slug: pkgdown-1-3-0
+description: > 
+  pkgdown 1.3.0 is now on CRAN!
+categories:
+  - package
+photo:
+  url: https://pixabay.com/en/decoration-packages-gifts-ribbons-3229259/
+  author: jackmac34
+---
+
+
+
+We're happy to announce that [pkgdown](https://pkgdown.r-lib.org/) 1.3.0 is now
+available on CRAN. pkgdown is designed to make it quick and easy to build a
+website for your package. Here, we'll highlight some of the new features, and
```

Comment:
We need something like (This blog post describes pkgdown 1.2.0 and 1.3.0 because we accidentally released 1.3.0 instead of 1.2.1)